cinema/tests/test_movie_session_api.py

```diff
@@ -0,0 +1,172 @@
+from django.test import TestCase
+from django.contrib.auth import get_user_model
+
+from rest_framework.test import APIClient
+from rest_framework.reverse import reverse
+from rest_framework import status
+
+from cinema.models import Movie, Genre, Actor
+from cinema.serializers import MovieListSerializer, MovieDetailSerializer
+
+MOVIE_URL = reverse("cinema:movie-list")
+
+
+def detail_url(movie_id: int) -> str:
+    return reverse("cinema:movie-detail", args=(movie_id,))
+
+
+def sample_movie(**params) -> Movie:
+    defaults = {
+        "title": "Groundhog Day",
+        "description": "A narcissistic, self-centered weatherman finds himself in a time loop on Groundhog Day.",
+        "duration": "101",
```

Comment:
The `duration` field in the `sample_movie` function is set as a string ('101'), but it should be an integer to match the model's field type. Consider changing it to an integer.